src/serlo-editor/plugins/input-exercise/index.ts

```diff
@@ -33,22 +32,20 @@ export type InputExercisePluginState = ReturnType<
 >
 
 export interface InputExerciseConfig {
-  feedback?: ChildStateTypeConfig
+  feedback: ChildStateTypeConfig
 }
 
 export type InputExerciseProps = EditorPluginProps<
   InputExercisePluginState,
   InputExerciseConfig
 >
 
-const defaultFeedback = { plugin: EditorPluginType.Text }
```

Comment:
I'd keep the default so other integrations do not have to provide something here (thinking of edusharing)

Reply:
Brought it back!